.github/workflows/nightly-release.yml

```diff
@@ -0,0 +1,111 @@
+# This GitHub action triggers a fresh set of Nomad builds
+# and publishes them to GitHub Releases under the `nightly` tag.
+# Note that artifacts available via GitHub Releases are not codesigned or notarized.
+# Failures are reported to slack.
+#
+
+name: Nightly Release
+
+on:
+  # TODO: remove the push event before merging this PR; this is for testing purposes
+  schedule:
+    # Runs against the default branch every day at midnight
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  # Build a fresh set of artifacts
+  build-artifacts:
+    # TODO: change branch from add-nightlies to main before merging this PR
+    uses: hashicorp/nomad/.github/workflows/build.yml@add-nightlies
```

Comment:
Ah we'll need to switch this back to main!

Reply:
Great eye! Thank you @sarahethompson ❤️